atlas/template_config.yaml

```diff
@@ -217,7 +217,6 @@ diamond_gap_extend: 1
 diamond_block_size: 2
 diamond_index_chunks: 4
 run_prokka: false
-eggnogg_dir: /path/to/databases
```

Comment:
This is a separate issue, but I also just noticed this in the config file. It appears to no longer be needed, now that the Snakefile (line 169) determines the eggnog directory from the database directory path:
```
EGGNOG_DIR = os.path.realpath(config["database_dir"])
```